src/EnergyPlus/api/TypeDefs.h

```diff
@@ -51,4 +51,18 @@
 typedef float Real32;  // Platform-specific: C++ has no defined precision floating point types
 typedef double Real64; // Platform-specific: C++ has no defined precision floating point types
 
+#ifdef __cplusplus
+namespace EnergyPlus {
+
+enum class Error: int {
+  Continue = 0,
+  Info = 1,
+  Warning = 2,
+  Severe = 3,
+  Fatal = 4
+};
+
+} // namespace EnergyPlus
+#endif
```

Comment:
So if you are accessing this endpoint in C, or Python, you'll just need to hardcode the numbers?  I do like the use of a proper enum class inside C++, but what are your thoughts on balancing that nicety vs having to hardwire numbers elsewhere.  If we went with a plain C enum, then we could access it nicely through the API.

Reply:
Yeah this certainly does prioritize C++, which to me is a feature not a bug but that is subjective. The C api is just an int https://github.com/NREL/EnergyPlus/blob/8093-error-callback/src/EnergyPlus/api/func.h#L94 

A C style enum would be ok, but what would the enum values be named? There are no namespaces so you would probably want something like ENERGYPLUS_ERROR_CONTINUE, or something like that. Perhaps with less uppercase shouting.

@lefticus you got any input?